giraffez/commandline.py

```diff
@@ -322,8 +322,8 @@ def run(self, args):
                 if f.file_type == GIRAFFE_ARCHIVE_FILE:
                     encoder = _encoder.Encoder(columns)
                     processors.append(encoder.unpack_row)
-                else:
-                    processors.append(text_to_strings(src_delimiter))
+                else: # text_to_string is handled by the csv parser
```

Comment:
From what I remember we can probably eliminate the `else` here, but I'm not 100% sure there isn't something that needs to be in this branch